src/components/organisms/events/table/row.tsx

```diff
@@ -18,18 +18,18 @@ export const EventRow = memo(
 		onClick: () => void;
 		style: CSSProperties;
 	}) => (
-		<Tr className="cursor-pointer pl-3 hover:bg-gray-750" onClick={onClick} style={style}>
-			<Td className="w-1/4" title={moment(createdAt).local().format(dateTimeFormat)}>
+		<Tr className="cursor-pointer hover:bg-gray-750" onClick={onClick} style={style}>
+			<Td className="mr-2 w-36 pl-4" title={moment(createdAt).local().format(dateTimeFormat)}>
 				{moment(createdAt).local().format(dateTimeFormat)}
 			</Td>
-			<Td className="w-1/4" title={eventId}>
+			<Td className="mr-2 w-24" title={eventId}>
 				<IdCopyButton id={eventId} />
 			</Td>
-			<Td className="w-1/4" title={destinationId || ""}>
+			<Td className="mr-2 w-24" title={destinationId || ""}>
 				<IdCopyButton id={destinationId || ""} />
 			</Td>
 			<Td className="w-1/4" title={eventType}>
-				{eventType}
+				<div className="truncate">{eventType}</div>
```

Comment:
When this will be merged: https://github.com/autokitteh/web-platform/pull/761

Remove className="truncate":
```suggestion
				{eventType}
```